.gitignore

```diff
@@ -164,3 +164,6 @@ modules.xml
 
 # End of https://www.toptal.com/developers/gitignore/api/flutter,intellij+all
 .run/
+
+.env
+.env.template
```

Comment:
I think the file `.env.template` shouldn't be in the gitignore, because it will be needed at the installation by dev, and we want it to be updatable in the long run.

Reply:
actually, I added it because while I was explaining, I saw that it was possible to add the token in this file without replacing the name of the file and commit it by mistake. I added .env.template to prevent mistake commit ;)

Reply:
Okayy well if you think it's best that way, let's go with that 😄